src/SFA.DAS.EAS.Api/SFA.DAS.EAS.Account.Api.csproj

```diff
@@ -393,6 +394,14 @@
       <Project>{8afb0bbe-1e53-488c-a971-d236df9eaa77}</Project>
       <Name>SFA.DAS.EAS.Account.Api.Types</Name>
     </ProjectReference>
+    <ProjectReference Include="..\SFA.DAS.EmployerAccounts.Api.Client\SFA.DAS.EmployerAccounts.Api.Client.csproj">
```

Comment:
Can these be nuget references?

Reply:
Yeah, they'll need to be. I'll delay changing them until you've answered the comment above though.